src/Native/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -238,7 +238,17 @@ C_FUNC(\Name):
 #define THREAD__VALIDATEEXINFOPOP C_FUNC(_ZN6Thread17ValidateExInfoPopEP6ExInfoPv)
 
 .macro INLINE_GETTHREAD
-        call    C_FUNC(RhpGetThread)
+        // Inlined version of call C_FUNC(RhpGetThread)
+       .att_syntax
+#if defined(__APPLE__)
+        movq    _tls_CurrentThread@TLVP(%rip), %rdi
+        callq   *(%rdi)
+#else
+        leaq    tls_CurrentThread@TLSLD(%rip), %rdi
```

Comment:
This means that we are hardcoding support of only the Local Dynamic TLS model. Since the runtime is used as a static lib that can be linked into any kind of executable, I wonder how to support the other models, e.g. the Initial Exe one. Maybe we should later build separate version of the static lib for each (or subset of) model?